primitives/ethereum-poa/src/lib.rs

```diff
@@ -83,6 +88,21 @@ pub struct Header {
 	pub seal: Vec<Bytes>,
 }
 
+/// Parsed ethereum transaction.
+#[derive(Debug, PartialEq)]
+pub struct Transaction {
```

Comment:
There's a lot of such crates, but most lack `no_std` support and others lack something we need :/ We can 'fix' some crates first, or make functional bridge, gather list of all changes we need to make to these crates, made these changes and update bridge to use these crates. I'd vote for bridge-first, everything other later. We can choose the opposite option, if you think it'd be better.

Reply:
If it doesn't take a long time to make these crates `no_std` compatible I think we should try and do that from the get-go. Otherwise, I'd be okay updating the bridge at a later time.

Reply:
Im fine with ditching other libraries and having the code duplicated in the repo. Can we make it into it's own crate though? It's going to be useful not only for the PoA use case.

Reply:
It is already in separate crate, but this PR adds exchange primitives to that crate => I've extracted it to separate crate. The name of eth/poa-related crate is still `sp-bridge-eth-poa`, though